10_msi_prediction/run_msi_lasso_range.sh

```diff
@@ -3,44 +3,49 @@
 # Run feature selection experiments across a few genes, with selected cancer
 # types held out (either partially or completely)
 
-RESULTS_DIR=./10_msi_prediction/results/msi_lasso_range
+RESULTS_DIR=./10_msi_prediction/results/msi_lasso_range_sex_covariate_lr
 ERRORS_DIR=./msi_lasso_range_errors
 
 mkdir -p $ERRORS_DIR
 
 lasso_penalties=(
-  "0.00000001"
-  "0.0000001"
-  "0.000001"
-  "0.00001"
-  "0.0001"
-  "0.0005"
   "0.001"
   "0.005"
   "0.01"
   "0.05"
   "0.1"
+  "1"
+  "10"
+  "100"
+  "250"
+  "500"
+  "750"
+  "1000"
+  "1500"
+  "2000"
 )
 
 for seed in 42 1; do
 
     for lasso_penalty in "${lasso_penalties[@]}"; do
 
-        cmd="python 10_msi_prediction/run_msi_prediction.py "
-        cmd+="--lasso_penalty $lasso_penalty "
-        cmd+="--num_features 16148 "
-        cmd+="--results_dir $RESULTS_DIR "
-        cmd+="--seed $seed "
-        cmd+="--training_samples pancancer "
-        cmd+="2>$ERRORS_DIR/errors_${seed}_${lasso_penalty}_pancancer.txt"
-        echo "Running: $cmd"
-        eval $cmd
+        # cmd="python 10_msi_prediction/run_msi_prediction.py "
```

Comment:
Do you want to keep this commented code in?

Reply:
Nope, it should be fine to get rid of it.